src/modules/aws_ec2/migration/1674575646446-aws_ec2.ts

```diff
@@ -0,0 +1,165 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
Do we need this migration change?

Reply:
We shouldn't, anymore. Probably easier to just `git checkout main -- src/modules/aws_ec2` to undo all of this.